tools/site_admin/manage_special_days.php

```diff
@@ -299,6 +299,10 @@ function _show_edit_row($field, $label, $type='text', $maxlength=null, $min=null
         {
             $addl_data = "<br>" . sprintf(_("See the full list of emojis in <a href='%s'>the Unicode standard</a>."), "http://unicode.org/emoji/charts/full-emoji-list.html");
         }
+        elseif($field == "spec_code")
+        {
+            $addl_data = "<br>" . _("This string is used as the primary key in the DB and cannot be changed after created.");
```

Comment:
Maybe O.T. but should we say the code should not be like "Birthday nnnn" or "Otherday nnnn" because these strings have a special meaning. Or perhaps also validate to exclude these.

Reply:
We probably ought to but I don't really want to invest the effort. This is rare enough that figuring out how and where to explain it seems difficult and not worth the edgecase.

Reply:
If it needs to be documented, it can be documented for the admins separately, as they are the only ones who can add/should be adding new special days.